aliasing/api/combat.py

```diff
@@ -333,6 +334,7 @@ def __init__(self, group: CombatantGroup):
         self._group = group
         self.type = "group"
         self.combatants = [SimpleCombatant(c) for c in self._group.get_combatants()]
+        self.name = group.name
```

Comment:
whoops, already implemented this in `2020w42` branch - remove to prevent merge conflict